docs/packaging/updating-an-existing-package.md

```diff
@@ -37,8 +38,8 @@ If you do not yet have a clone of the package, change to your packaging director
 
```

Comment:
On the previous line 
"If you do not yet have a clone of the package..."

should be
"If you do not yet have a clone of your packages repo fork, change to your packaging directory and clone your fork. Then, switch to the  directory of the package to update. For example:"